### PR TITLE
securitywiki EFS security group updates

### PR DESCRIPTION
**Pull request by cmharlow**
Jira: https://mozilla-hub.atlassian.net/browse/SE-2159

What this PR does:
* updates SecurityWiki EFS to be in the primary eks security group, not the ("additional") eks security group
* fixes SecurityWiki pods talking to EFS over a NFS network connection

Note:
* came up in repairing this - the manually added old security group ids will be removed once this app is migrated
* tagging @duallain on this since he paired with me on it, but happy to walk anyone through what is going on here

### CODE REVIEW

**Comment by cmharlow** [comment]
also sorry, i named this branch poorly

**Review by duallain — APPROVED** [review]
LGTM, though we paired on the work, so up to you if you want to wait on a second review

**Pull request merged**
